forest-node/ansible/roles/forest/tasks/main.yml

```diff
@@ -0,0 +1,62 @@
+---
+# tasks file for role/forest
+- name: Upgrade all apt packages
+  apt: upgrade=dist force_apt_get=yes
```

Comment:
Is it necessary to upgrade the packages?

Reply:
Yes actually - this is synonymous with `sudo apt-get update`

Reply:
Why is it necessary?

Reply:
This is a server just created and normal practice is to update which simply makes sure our list of packages from all repositories and PPAs is up to date. If we do not run this command, we could be getting older versions of various packages we install, or worse, dependency issues. 

This is just good practice when interacting with Linux servers

Reply:
Whether this is good practice is highly debated. Nix, for example, has the complete opposite point of view and I'm quite partial to this way of seeing things. If there's a security vulnerability in the image we're using then we'll update the image rather updating the software in-place.

Reply:
Well, afaik, I am not aware of any vulnerability - but I'd try the run the play now w/o any of the package update process and see how it goes

Reply:
Just tested this and it works w/o updating packages so i'd push the changes